lib/rust/lexer/generation/tests/enso_lexer_blocks.rs

```diff
@@ -0,0 +1,337 @@
+#![feature(test)]
+#![deny(unconditional_recursion)]
+#![warn(missing_copy_implementations)]
```

Comment:
lets do macros for these tests - talk with me and @mwu-tow  about their design. Now its really hard to read these

Reply:
Filed as #1247.